packages/app/prisma-utils/src/prismaTransaction.ts

```diff
@@ -104,11 +111,17 @@ const calculateRetryDelay = (
 	return Math.min(expDelay, maxDelayMs)
 }
 
-const isRetryAllowed = <T>(result: PrismaTransactionReturnType<T>, dbDriver: DbDriver): boolean => {
+type isRetryAllowedResult = boolean | 'increase-timeout'
+
```

Comment:
@kjamrog I implemented your suggestion at the beginning, but including the object for this only reason sounds to me like too much, but if you feel it is really better let me know and I will change it

Reply:
Let's leave it as it is then, no worries!